code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1923,3 +1923,18 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 		runechat_msg_location = loc
 	else
 		runechat_msg_location = src
+
+/mob/living/carbon/human/verb/Examine_OOC()
```

Comment:
Moved here and made into a verb, since this never worked before due to it being `/mob/living/proc/examine_OOC()`.